post-processor/vsphere/post-processor.go

```diff
@@ -58,13 +71,17 @@ func (p *PostProcessor) Configure(raws ...interface{}) error {
 
 	// Defaults
 	if p.config.DiskMode == "" {
-		p.config.DiskMode = "thick"
+		p.config.DiskMode = "thin"
```

Comment:
what's the effect of this change?

Reply:
Thank you for your prompt response. This change is useful for example, when building VMs in remote nested esxi host with limited storage capabilities. By marking the disk type as thin, we force the use of a sparse rather a raw disk image. It can be handy sometimes.

Reply:
Do you think that change belongs in this patch? Is "thin" a reasonable default?

Reply:
I think this change, as not belonging to this patch (my bad) but I consider it as a reasonable default for three reasons:
- time, a raw image would take longer to build, can be cloned faster.
- space, thin takes less space than raw.
- flexibility, it can be transformed to raw, while deploying new vm from that template for example.

Reply:
I'd like to keep the default the same.